docs/src/account/address.md

```diff
@@ -6,20 +6,35 @@
 
 ## Purpose
 
-An address is an identifier that facilitates sending and receiving of [notes](../note.md). It has essentially two purposes that are explained in this section.
+An address is an identifier that facilitates sending and receiving of [notes](../note.md). It serves three main purposes explained in this section.
 
-### Access control for notes
+### Communicating receiver information
 
-An address expresses who is allowed to consume a note. In this sense, it acts as an access-control mechanism.
+An address is designed for the note receiver to communicate information about themselves to the sender.
 
-Consider two examples that use different access control mechanisms:
+The receiver can choose to disclose various pieces of information, which control both how the note itself is structured and how the note is to be communicated to the receiver.
```

Comment:
> how the note is to be communicated to the receiver.

I don't quite understand what this is supposed to say _in this section_. Is this alluding to the "communicating channel information" section, i.e. to tell the sender over which channel to send the note? If so, I would move this to that section.